TRAVIS/build_total.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+import re
+import requests
+import json
+import os
+import sys
+import subprocess
+
+
+
+class build_class():
+
+    def __init__(self, rep, basedir):
+        if (rep == 'ecl' or rep == 'res' or rep == 'ert'):
+            self.repository = rep
+            if (rep == 'ecl'):
+                self.rep_name = 'libecl'
+            if (rep == 'res'):
+                self.rep_name = 'libres'
+            if (rep == 'ert'):
+                self.rep_name = 'ert'
+            self.ecl_pr_num = -1
+            self.res_pr_num = -1
+            self.ert_pr_num = -1
+            self.pr_description = None
```

Comment:
No reason to internalize this as a member in the class - just pass `content['body']` to the parse routine.